### PR TITLE
241 add loader

### PR DESCRIPTION
**Pull request by Shushunya**
image

### CODE REVIEW

File: FrontEnd/src/components/loader/Loader.jsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import css from "./Loader.module.css";
+import { Carousel, ConfigProvider } from "antd";
+
+export default function Loader() {
+  return (
+    <div className={css.container}>
+      <ConfigProvider
```
Comment (olehpopovych-amplify) [inline FrontEnd/src/components/loader/Loader.jsx:7]: I think config provide should be moved to app root with rest providers

File: FrontEnd/src/components/loader/Loader.jsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
+import css from "./Loader.module.css";
```
Comment (olehpopovych-amplify) [inline FrontEnd/src/components/loader/Loader.jsx:1]: Could we add a loader to the page where we fetch regions?
Also, I would really appreciate it if you add some screenshots to the pr description

Comment (Shushunya) [reply FrontEnd/src/components/loader/Loader.jsx:1]: Screenshot added to the description of the PR :)

File: FrontEnd/src/components/loader/Loader.jsx [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+          </div>
+          <div>
+          <span/>
+          </div>
+          <div>
+          <span/>
+          </div>
+          <div>
+          <span/>
+          </div>
```
[the remark this answers is not in the document]
Comment (Shushunya) [reply FrontEnd/src/components/loader/Loader.jsx]: done

**Comment by Shushunya** [comment]
One test on the server side is still failing. But my PR was purely the client side part. Also I've changed 2 tests in authorization so now they also pass.

**Pull request merged**